pkg/worker/worker.go

```diff
@@ -214,6 +222,7 @@ func (w *Worker) setupGit(oldstatus bool, ex executor.Executor, repoDir string)
 func (w *Worker) setupTests(ex executor.Executor, workDir, repoDir string) (bool, error) {
```

Comment:
@mohammedzee1000 Can we also add more logs for remote fetch error https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/17283/rehearse-17283-pull-ci-openshift-odo-master-psi-kubernetes-integration-e2e/1376807362232324096#1:build-log.txt%3A95

Reply:
sure, let me see what i can do